### PR TITLE
Fixed #34311 -- Updated serialization docs from unique_together to UniqueConstraint.

### PR DESCRIPTION
**Pull request by KommuSoft**
Fixing ticket 34311: https://code.djangoproject.com/ticket/34311#ticket

### CODE REVIEW

**Review by emarhin — APPROVED** [review]

**Comment by felixxm** [comment]
@KommuSoft Thanks :+1:

I pushed small edits.

File: docs/topics/serialization.txt [hunk 1]
```diff
@@ -454,12 +464,12 @@ into the primary key of an actual ``Person`` object.
...
-    in your natural key. However, uniqueness doesn't need to be
-    enforced at the database level. If you are certain that a set of
-    fields will be effectively unique, you can still use those fields
-    as a natural key.
+    have a uniqueness clause (either ``unique=True`` on a single field, or
+    a ``UniqueConstraint`` or ``unique_together`` over multiple fields)
+    for the field or fields in your natural key. However, uniqueness
+    doesn't need to be enforced at the database level. If you are certain
+    that a set of fields will be effectively unique, you can still use
+    those fields as a natural key.
```
Comment (felixxm) [inline docs/topics/serialization.txt:472]: Wrap at 79 chars:
```suggestion
    have a uniqueness clause (either ``unique=True`` on a single field, or a
    ``UniqueConstraint`` or ``unique_together`` over multiple fields) for the
    field or fields in your natural key. However, uniqueness doesn't need to be
    enforced at the database level. If you are certain that a set of fields
    will be effectively unique, you can still use those fields as a natural
    key.
```

File: docs/topics/serialization.txt [hunk 2]
```diff
@@ -387,7 +387,12 @@ Consider the following two models::
         birthdate = models.DateField()
 
         class Meta:
-            unique_together = [['first_name', 'last_name']]
+            constraints = [
+                models.UniqueConstraint(
+                    fields=('first_name', 'last_name'),
+                    name='unique_name'
+                )
+            ]
```
Comment (felixxm) [inline docs/topics/serialization.txt:395]:
```suggestion
            constraints = [
                models.UniqueConstraint(
                    fields=["first_name", "last_name"],
                    name="unique_first_last_name",
                ),
            ]
```

File: docs/topics/serialization.txt [hunk 3]
```diff
@@ -431,7 +436,12 @@ name::
         objects = PersonManager()
 
         class Meta:
-            unique_together = [['first_name', 'last_name']]
+            constraints = [
+                models.UniqueConstraint(
+                    fields=('first_name', 'last_name'),
+                    name='unique_name'
+                )
+            ]
```
Comment (felixxm) [inline docs/topics/serialization.txt:444]:
```suggestion
            constraints = [
                models.UniqueConstraint(
                    fields=["first_name", "last_name"],
                    name="unique_first_last_name",
                ),
            ]
```

File: docs/topics/serialization.txt [hunk 4]
```diff
@@ -479,7 +489,12 @@ Firstly, you need to add another method -- this time to the model itself::
...
```
Comment (felixxm) [inline docs/topics/serialization.txt:497]:
```suggestion
            constraints = [
                models.UniqueConstraint(
                    fields=["first_name", "last_name"],
                    name="unique_first_last_name",
                ),
            ]
```

**Review by felixxm — COMMENTED** [review]
I cannot push small edits to your branch, so I left detailed suggestions.

**Comment by felixxm** [comment]
Updated in #16535.

**Pull request closed without merging**